server/db.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable sort-keys */
+/* eslint-disable no-multi-assign */
+// faux database
+
```

Comment:
there is no need to push these objects into the arrays. Just add the objects in the array as you are defining the array